bazel/cc_eventuals_library.bzl

```diff
@@ -6,5 +6,4 @@ load("@com_github_reboot_dev_pyprotoc_plugin//:rules.bzl", "create_protoc_plugin
 
 cc_eventuals_library = create_protoc_plugin_rule(
     "@com_github_3rdparty_eventuals//protoc-gen-eventuals:protoc-gen-eventuals",
-    extensions = (".eventuals.h", ".eventuals.cc"),
```

Comment:
I assume this relies on the `declare_directory` approach that we discussed? Did you get it to work and does it pick up changes automatically?